Makefile

```diff
@@ -5,38 +5,34 @@ CCFLAGS = -Wall -c
 
 ifeq ($(OS),Windows_NT)
 	os = windows
-	CCFLAGS += -D WIN32
+	CCFLAGS += -D WIN32 -D IA32
 	LDFLAGS += -l rasapi32 -l wininet
-	BIN = pac.exe
-	ifeq ($(PROCESSOR_ARCHITECTURE),AMD64)
-		CCFLAGS += -D AMD64
-	endif
-	ifeq ($(PROCESSOR_ARCHITECTURE),x86)
-		CCFLAGS += -D IA32
-	endif
+	BIN = binaries/windows/pac
```

Comment:
To help with go-bindata, and to help organize the binaries, I'm now putting them in subfolders.